### PR TITLE
Add principles on codes of conduct

### PR DESCRIPTION
**Pull request by johnwaterworth**
Add a new section on hosting events. Making them accessible, inclusive and safe. And having a code of conduct for every event. Add an expectation that staff attending events follow the code of conduct for the event.

### CODE REVIEW

File: _includes/sections/work-we-do/sharing-our-expertise.md [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
 
+#### Hosting events
+
+We host a variety of face to face and online events to share our experience and
+expertise.
+
+We work hard to design events that are accessible, inclusive and safe. We ask
+attendees about their individual access and support needs. We publish a code of
+conduct for every event we host, and require one from every event we are
+involved with.
```
Comment (erbridge) [inline _includes/sections/work-we-do/sharing-our-expertise.md:11]: > require one from every event we are involved with

Is this true, yet?

**Review by erbridge — APPROVED** [review]

**Comment by wendydxw** [comment]
Hi,

It isn't. We should point to the code we adopt in terms of our
expectations, and use our influence to actively encourage others to do the
same where they haven't already.

Wendy

Wendy Coello

Director Marketing and Comms, dxw

She/her

wendy@dxw.com

@coellowa <https://twitter.com/coellowa?lang=en>

07940 752337

www.dxw.com :: 0345 2577 520 :: @dxw <https://twitter.com/dxw?lang=en> ::
we create public services that improve lives

File: _includes/sections/work-we-do/sharing-our-expertise.md (hunk 1 above)
Comment (CristinaRO) [reply _includes/sections/work-we-do/sharing-our-expertise.md:11]: I take this to mean that it _will_ be true for every event dxw get involved with from now on.

**Review by CristinaRO — APPROVED** [review]
I

**Comment by wendydxw** [comment]
I don't think we can commit to that.


Wendy Coello

Director Marketing and Comms, dxw

She/her

wendy@dxw.com

@coellowa <https://twitter.com/coellowa?lang=en>

07940 752337

www.dxw.com :: 0345 2577 520 :: @dxw <https://twitter.com/dxw?lang=en> ::
we create public services that improve lives

File: _includes/sections/work-we-do/sharing-our-expertise.md (hunk 1 above)
Comment (CristinaRO) [reply _includes/sections/work-we-do/sharing-our-expertise.md:11]: Given Wendy's reply (https://github.com/dxw/playbook/pull/435#issuecomment-679083410), maybe we want to replace "require" with a less strong word.

Could we clarify what "involved with" means? If it's an event we sponsor, we could require a code of contact in a way we might not be able to if it's an event where we only participate by having someone speak or present.

**Comment by wendydxw** [comment]
What about something like:

dxw applies a code of conduct to all the events it hosts. We expect the
hosts of the external events we are involved in to apply similar standards
to the conduct of their sponsors, event organisers and participants.

That way we're not requiring external orgs to have a code, but we should
look to influence them by drawing attention to our code and always call out
bad behaviour.

W

Wendy Coello

Director Marketing and Comms, dxw

She/her

wendy@dxw.com

@coellowa <https://twitter.com/coellowa?lang=en>

07940 752337

www.dxw.com :: 0345 2577 520 :: @dxw <https://twitter.com/dxw?lang=en> ::
we create public services that improve lives

**Pull request merged**